wgsl/index.bs

```diff
@@ -9102,9 +9102,14 @@ struct __modf_result_vecN {
 
   <tr algorithm="smoothStep">
     <td>|T| is [FLOATING]
-    <td class="nowrap">`smoothStep(`|e1|`:` |T| `, `|e2|`:` |T| `, `|e3|`:` |T| `) -> ` |T|
+    <td class="nowrap">`smoothStep(`|low|`:` |T| `,` |high|`:` |T| `,` |x|`:` |T| `) -> ` |T|
     <td>Returns the smooth Hermite interpolation between 0 and 1.
     [=Component-wise=] when |T| is a vector.
+
+    For scalar |T|, the result is
+    |t| * |t| * (3.0 - 2.0 * |t|),
```

Comment:
should we use `&times;` and `&minus;` here?

Reply:
Good question.  We have not been consistent about this.  Sometimes the definition of a function is defined in terms of ideal math, and sometimes in terms of WGSL expressions.   
The accuracy rule for this one is "inherited from" so the strong suggestion is that it is literally implemented this way.
I'll file an issue to make a decision on this and be consistent.